brave/brave-core
# Honor BrowserGuestModeEnabled policy.

Resolves https://github.com/brave/brave-browser/issues/12524

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.

## Test Plan:

Assuming a Linux machine:

1. Create /etc/brave/policies/managed/policy.json with contents: {"BrowserAddPersonEnabled": false}
2. Confirm policy has effect using brave://policy
3. Open the "hamburger menu" and verify that user can not "create a new profile".
4. Create /etc/brave/policies/managed/policy.json with contents: {"BrowserGuestModeEnabled": false}
5. Open the "hamburger menu" and verify that user can not "open guest window".
